### DISCUSSION TITLE
Score-Based Diffusion Models | Fan Pu Zeng

### DISCUSSION THREAD

**Discussion by giscus** [fanpu/website#14] [General]
# Score-Based Diffusion Models | Fan Pu Zeng

Score-based diffusion models are a promising direction for generative models, as they improve on both likelihood-based approaches like variational autoencoders, as well as adversarial methods like Generative Adversarial Networks (GANs).  In this blog post, we survey recent developments in the field centered around the line of results developed in (Song & Ermon, 2019), analyze the current strengths and limitations of score-based diffusion models, and discuss possible future directions that can address its drawbacks. Joint work with Owen Wang.

https://fanpu.io/blog/2023/score-based-diffusion-models/

**Comment by fanpu** [comment]
Fixed a rendering issue that resulted due to a change in the build process - sorry about the previous confusion!
